content/bn/style-guide/_index.md

```diff
@@ -7,125 +7,121 @@ menu:
 
-The Glossary aims at **explaining complex concepts in simple words** — that is a surprisingly difficult task that will likely take multiple revisions. Always keep the audience in mind when drafting your definition. Avoid using industry terms and buzzwords — you'll probably catch yourself going back to them and may need to autocorrect. 
+আপনার সংজ্ঞায় ব্যবহৃত হলে, সর্বদা *বিদ্যমান শব্দকোষের শর্তাবলীর সাথে লিঙ্ক করুন* (শুধুমাত্র প্রথম উল্লেখ হাইপারলিঙ্ক করা উচিত)।
 
-When appropriate, use **real-world examples** that help readers (especially non-technical ones) better understand *when* and *why* the concept you’re explaining is relevant. 
+*উদাহরণ*: [পরিষেবা মেশ সংজ্ঞা](https://glossary.cncf.io/service_mesh/) এর “এটি কী” বিভাগটি একবার দেখুন। এটি মাইক্রোসার্ভিস, পরিষেবা, নির্ভরযোগ্যতা এবং পর্যবেক্ষণযোগ্যতার সংজ্ঞাগুলির সাথে লিঙ্ক করে। উপরন্তু, এটি একটি মাইক্রোসার্ভিসেস পরিবেশে নেটওয়ার্ক চ্যালেঞ্জের তুলনা করে একটি বাস্তব-বিশ্বের উদাহরণ ব্যবহার করে (এমন কিছু যা অ-প্রযুক্তিগত লোকেরা সম্পর্কিত হতে পারে না)  ওয়াইফাই সমস্যার (যা কেউ ল্যাপটপ ব্যবহার করে বুঝতে পারে)সাথে । যেখানে সম্ভব, সেই সংযোগটি তৈরি করার চেষ্টা করুন।
 
-When used in your definition, always **link to existing glossary terms** (only the first mention should be hyperlinked).
 
-**Example**: take a look at the “What it is” section of the [service mesh definition](https://glossary.cncf.io/service_mesh/). It links back to the microservices, service, reliability, and observability definitions. Additionally, it uses a real-world example comparing network challenges in a microservices environment (something non-technical people can't relate to) to wifi problems (something anyone using a laptop can understand). Where possible, try to make that connection. 
 
-#### Start with a Google or Word doc
+#### একটি Google বা Word ডক দিয়ে শুরু করুন
 
-We recommend starting with a Google or Word doc, letting it sit for a few days, and revisiting again. This will allow you to catch phrases or expressions that could be worded in a simpler and more accessible way. Also, make sure to run a spellcheck before submitting a PR.
+আমরা একটি Google বা Word ডক দিয়ে শুরু করার পরামর্শ দিই, এটিকে কয়েক দিনের জন্য বসতে দিন এবং আবার দেখার জন্য। এটি আপনাকে বাক্যাংশ বা অভিব্যক্তিগুলি ধরতে দেয় যা একটি সহজ এবং আরও অ্যাক্সেসযোগ্য উপায়ে শব্দ করা যেতে পারে। এছাড়াও,  PR জমা দেওয়ার আগে একটি বানান পরীক্ষা চালানো নিশ্চিত করুন।
 
-To ensure no one else submits a PR while working on a term, make sure to claim an issue (or create one) and that it is assigned to you.  More to that in the [How To Contribute](https://glossary.cncf.io/contribute/) doc.
+একটি মেয়াদে কাজ করার সময় অন্য কেউ PR জমা না দেয় তা নিশ্চিত করতে, একটি সমস্যা দাবি করা (বা একটি তৈরি করুন) এবং এটি আপনাকে বরাদ্দ করা হয়েছে তা নিশ্চিত করুন। [কীভাবে অবদান রাখতে হয়](https://glossary.cncf.io/contribute/) ডক-এ আরও কিছু আছে যা দেখতে পারেন  ।
```

Comment:
Let's update links to the localized version if exists:

Replace `https://glossary.cncf.io/contribute/` → `/bn/contribute/`